interfaces/Portal/src/app/models/payment.model.ts

```diff
@@ -12,7 +12,7 @@ export class Payment extends PaymentData {
 }
 
 export class LastPaymentResults {
-  error: number;
+  failed: number;
```

Comment:
I wasn't sure about this, so if we would like to keep 'error' we can revert last commit